Jetsnack/app/src/main/java/com/example/jetsnack/ui/utils/SystemUi.kt

```diff
@@ -19,12 +19,90 @@ package com.example.jetsnack.ui.utils
 import android.os.Build
 import android.view.View
 import android.view.Window
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.Providers
+import androidx.compose.runtime.remember
 import androidx.compose.runtime.staticAmbientOf
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.compositeOver
 import androidx.compose.ui.graphics.luminance
 import androidx.compose.ui.graphics.toArgb
 
+private val AmbientStatusBar = staticAmbientOf { UnspecifiedSystemBar }
+
+private val AmbientNavigationBar = staticAmbientOf { UnspecifiedSystemBar }
+
+@Composable
+fun StatusBar(
+    color: Color,
+    darkIcons: Boolean = color.luminance() > 0.5f,
+    transformColorForLightContent: (Color) -> Color = BlackScrimmed,
+    content: @Composable () -> Unit
+) {
+    val statusBar = SystemBar(color, darkIcons, transformColorForLightContent)
+    val controller = AmbientSysUiController.current
+    val status = AmbientStatusBar.current
+    val prevStatus = remember { status }
+    DisposableEffect(statusBar, prevStatus) {
```

Comment:
Remove `prevStatus` , as the value will never change

Reply:
Done for both